Homework3/Task02/test02.py

```diff
@@ -0,0 +1,10 @@
+from time import perf_counter
+
+from Task02.task02 import fast_calculate
+
+
+def test_fast_calculate():
```

Comment:
Please don't test the time, test the result.

Reply:
What do you mean? I thought the task was about reducing running time. Should I check that the answer is the same? But what about time and 1 min time limit?

Reply:
I mean this:

> Calculate total sum of slow_calculate() of all numbers starting from 0 to 500. 

Have you done it?  Is it correct?